### PR TITLE
Update code to work for move from CyHy to the COOL

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This PR involves URL changes and the addition of an AWS profile to use with the COOL account.

## 💭 Motivation and Context

The DNS account in the COOL is the only account where the `dmarc.cyber.dhs.gov` domain is verified.  Therefore that is where the DMARC aggregate report emails are sent, and so the dmarc-import application (including the Elasticsearch DB) must be moved there.

## 🧪 Testing

I am currently using these changes to copy over the ES database.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: copy_old_es_database.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 from requests_aws4auth import AWS4Auth
 
 OLD_AWS_PROFILE_NAME = 'default'
+NEW_AWS_PROFILE_NAME = 'cool-dns-amazonesfullaccess'
```
Comment (dav3r) [inline copy_old_es_database.py:9]: Do we need to include this profile in our set of profiles used with [`aws-profile-sync`](https://github.com/cisagov/aws-profile-sync)?

**Review by dav3r — APPROVED** [review]
LGTM! 🏭

File: copy_old_es_database.py (hunk 1 above)
Comment (jsf9k) [reply copy_old_es_database.py:9]: No, it's a temporary one I created to give me temporary permissions to do whatever I wanted to the DB.  Normally access is granted via Cognito and a role associated with that.  That is the bit that I hand-jammed before, but I have a note to revisit and code up correctly in the COOL.

Comment (dav3r) [reply copy_old_es_database.py:9]: That's what I figured, but just wanted to confirm. 👍

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Pull request merged**